spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -60,6 +60,16 @@ fun calculatePackageVerificationCode(files: List<File>) =
         })
     }
 
+/**
+ * Calculate the [SPDX package verification code][1] for all files in a [directory]. If [directory] points to a file
+ * instead of a directory the verification code for the single file is returned.
+ *
+ * [1]: https://spdx.org/spdx_specification_2_0_html#h.2p2csry
+ */
+@JvmName("calculatePackageVerificationCodeForDirectory")
+fun calculatePackageVerificationCode(directory: File) =
```

Comment:
Can we please add a test also for this variant to `SpdxUtilsTest`?

Reply:
Done.